### PR TITLE
Add Marathon watcher

### PR DESCRIPTION
**Pull request by ktf**
Fetches metrics from /metrics endpoint in
[Marathon](https://mesosphere.github.io/marathon/) and pushes them to
riemann. This is derived from the elasticsearch watcher.

### CODE REVIEW

**Comment by aphyr** [comment]
Seems legit! Thanks ktf. :)

**Comment by ktf** [comment]
Thanks. I have a problem with riemann-dash though. It seems to choke whenever I've more that a few dozens of metrics being displayed. Is that a known issue?

**Comment by aphyr** [comment]
I've pushed a few thousand events/sec through the dash, but, as you probably already know, computers. ;-)

**Comment by ktf** [comment]
Let me rephrase is. Is there any known slugginesh issue (i.e. it freezes) whenever you use the log / list views with more than a few dozens entries?

**Comment by aphyr** [comment]
I don't recall anyone else reporting this problem, but it obviously exists, since it's happening to you. Fire up a profiler?

**Comment by ktf** [comment]
Okok, I'll try to investigate more. BTW, there is also #128 which is for a related (but actually different) service.
